lib/index.js

```diff
@@ -47,6 +47,10 @@ const exceptionWrapper = function tryCatchWrapper(funcToWrap) {
       process.exit(0);
     } catch (err) {
       logger.error(err.message);
+      if (process.env.BINARIS_LOG_LEVEL === 'info' || !process.env.BINARIS_LOG_LEVEL) {
```

Comment:
This is odd.  E.g. it logs if I set `BINARIS_LOG_LEVEL=info`, but _not_ if i set `BINARIS_LOG_LEVEL=warn` or `BINARIS_LOG_LEVEL=quux`.

You definitely want to log if `!process.env.BINARIS_LOG_LEVEL`.  And arguably you might also want to log this if `!logger.isDebugEnabled` (or somesuch; also this works in Winston v3 I think but I dunno about v2).

Reply:
I don't understand. It's not disabling logging. It's just showing you a tip about logging levels if the user doesn't have a log level defined or if INFO is set.

Reply:
Ok that's more direct feedback. I don't mind removing the `info` case.